### PR TITLE
Make import error message more specific

### PR DESCRIPTION
**Pull request by nelson-liu**

### CODE REVIEW

File: codalab/worker_manager/gcp_batch_worker_manager.py [hunk 1]
```diff
@@ -1,6 +1,13 @@
 try:
     from google.cloud.container_v1 import ClusterManagerClient  # type: ignore
     from google.oauth2 import service_account  # type: ignore
+except ModuleNotFoundError:
+    raise ModuleNotFoundError(
+        'Running the worker manager requires the kubernetes module.\n'
```
Comment (epicfaace) [inline codalab/worker_manager/gcp_batch_worker_manager.py]:
```suggestion
        'Running the worker manager requires the google-cloud-container module.\n'
```

File: codalab/worker_manager/gcp_batch_worker_manager.py [hunk 2]
```diff
@@ -1,6 +1,13 @@
 try:
     from google.cloud.container_v1 import ClusterManagerClient  # type: ignore
     from google.oauth2 import service_account  # type: ignore
```
Comment (epicfaace) [inline codalab/worker_manager/gcp_batch_worker_manager.py:3]: Does running `pip install google-cloud-container` also install the required library for `google.oauth2`?

**Review by epicfaace — APPROVED** [review]

**Comment by epicfaace** [comment]
@nelson-liu ?

File: codalab/worker_manager/gcp_batch_worker_manager.py (hunk 2 above)
Comment (nelson-liu) [reply codalab/worker_manager/gcp_batch_worker_manager.py:3]: yup it does

**Comment by nelson-liu** [comment]
thanks for the ping @epicfaace

**Pull request merged**